templates/firefox-ios.xliff

```diff
@@ -4270,6 +4322,11 @@ Private Tab</source>
         <source>Show products recommended by %@</source>
         <note>Label of the switch from settings card displayed in the shopping review quality bottom sheet. The placeholder will be replaced with the app name.</note>
       </trans-unit>
+      <trans-unit id="Shopping.SettingsCard.SwitchValueOn.AccessibilityLabel.v123" xml:space="preserve">
```

Comment:
Nit: seems like there are two contradicting comments for this string (and that they are more developer comments vs l10n)?
Also, should there be a string for "Off" as well? seems like that is missing

Reply:
@thatswinnie halp!

Reply:
@Delphine looks like the string for the `off` was added under the same key as the `on`. I will let you know when this is fixed.

Reply:
@Delphine the PR to fix the key was just merged. Could you pull the strings again?

Reply:
Thanks for the fix Winnie!
I'll go ahead and merge this now.
Something to keep in mind for future instances: it would be appreciated if we could avoid using technical jargon like "bottom sheet" in localization comments. Using more user-friendly terms in l10n comments will be more beneficial, so localizers understand the full context. Thank you!